release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3326,6 +3326,62 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, when a pod specified a node selector with no matching nodes, the kube-scheduler panicked with the following error: _Observed a panic: integer divide by zero_. With this release, the issue in the kube-scheduler code base is resolved, and the kube-scheduler no longer panics when a pod specifies a node selector with no matching nodes. (link:https://issues.redhat.com/browse/OCPBUGS-36397[*OCPBUGS-36397*])
+
+* Previously, installing an Operator could sometimes fail if the same Operator had been previously installed and uninstalled. This was due to a caching issue. With this release, {olm} is updated to install the Operator correctly in this scenario and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-36452[*OCPBUGS-36452*])
+
+* Previously, the Ingress Operator could not successfully update the canary route because the Operator did not have permission to update `spec.host` or `spec.subdomain` on an existing route. With this release, the required permission is added to the cluster role for the Operator's ServiceAccount and the Ingress Operator can update the canary route. (link:https://issues.redhat.com/browse/OCPBUGS-36467[*OCPBUGS-36467*])
+
+* Previously, when the OVN-Kubernetes setting for routing-via-host was set to the default value of shared gateway mode, OVN-Kubernetes did not correctly handle traffic streams that mixed non-fragmented and fragmented packets from the IP layer on cluster ingress.  With this release, OVN-Kubernetes correctly reassembles and handles external traffic IP packet fragments when ingressing and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-36554[*OCPBUGS-36554*])
+
+* Previously, in Amazon Web Services (AWS) security token service (STS), Cloud Credential Operator (CCO) checked `awsSTSIAMRoleARN` in `CredentialsRequest` to create the secret. When `awsSTSIAMRoleARN` was not present, CCO logged the error. With this release, CCO no longer logs the error and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-36716[*OCPBUGS-36716*])
+
+* Previously, the Open vSwitch (OVS) pinning procedure set the CPU affinity of the main thread, but other CPU threads did not pick up this affinity if they had already been created. As a consequence, some OVS threads did not run on the correct CPU set and could interfere with the performance of pods with a Quality of Service (QoS) class of `Guaranteed`. With this update, the OVS pinning procedure updates the affinity of each OVS thread, ensuring that all OVS threads run on the correct CPU set. (link:https://issues.redhat.com/browse/OCPBUGS-37197[*OCPBUGS-37197*])
+
+[id="ocp-4-14-34-known-issue_{context}"]
+==== Known issues
+
+* On clusters with the SR-IOV Network Operator installed and configured, pods with a secondary interface of SRI-OV VF will fail to create a pod sandbox and thus will not function. (link:https://issues.redhat.com/browse/OCPBUGS-38091[*OCPBUGS-38091*])
```

Comment:
Suggestion to get rid of a bit of future tense. Could maybe do more, but here's at least a bit

```suggestion
* On clusters with the SR-IOV Network Operator installed and configured, pods with a secondary interface of SRI-OV VF fail to create a pod sandbox and will not function. (link:https://issues.redhat.com/browse/OCPBUGS-38091[*OCPBUGS-38091*])
```